src/reducers/consultant-reducers.ts

```diff
@@ -0,0 +1,11 @@
+import { ConsultantModel } from "../components/consultant/models";
+import { ACTION_TYPES } from "../actions";
+
+export const consultants = (state: ConsultantModel[] = [], action): ConsultantModel[] => {
+  switch (action.type) {
+    case ACTION_TYPES.CONSULTANTS_FETCHED:
+      return action.consultants
+    default:
+      return state
```

Comment:
Als er een nieuwe consultant toegevoegd wordt, komt deze dan ook direct in de store terecht?
Lijkt dat je hier de CONSULTANTS_UPDATED ook moet hebben?

Reply:
Nope momenteel niet.

Reply:
Deze changes zullen in de nieuwe pull request zitten (feature/create_project). Dus je kan deze pull request mergen.